### ISSUE TITLE
Font seems to load, but is not adding any boldness in browser, terminal, but does on abiword

### ISSUE THREAD

**Issue by khlsvr** [Born2Root/Fast-Font#18]
Hi.

I wonder what my problem is. I installed all the 3 fonts to my Arch Linux through ~/.local/share/fonts/ dir and they show up in `fc-list`. I can also load them in qutebrowser or firefox, and I can see a difference to the font I have before loading them at least on qb. However, they just change the outlook, but do not add boldness to the characters at the begin of words. Same goes for kitty terminal when I try run kitty with `kitty --override font_family='Fast_Sans'` I don't see any boldness. In fact, on kitty the font may not change at all. Tried with the dotted font and no dots are added. I am succesfully loading my custom conhanzi font on qutebrowser and it also has a lot of ligature substitutions going on, so there should be ligature support there. Also kitty supports ligatures and I can load conhanzi with the override flag succesfully there.

Then running abiword and picking the Fast_Sans from the font menu I can see it working properly bolding first chars on each words so the font files themselves should be alright I guess

edit: I might have figured this out. In firefox if I pick one of the fonts as the "monospace" font, then it works there properly. Maybe it has to do with needing a monospace variation and I just need to find out how I can force one of these to be forced as monospace on terminal and qutebrowser. However it doesn't explain why on qb I can see the font appearance changing when I switch :thinking: 

edit2: Kitty doesn't support non-monospaced fonts, so I guess I would need to create a monospaced variation of the font using your python script or something else I guess

**Comment by khlsvr** [comment]
Hey would it be possible for you or someone on windows computer to create a fast font from some Monospace font to add compatibility? For example DejaVu Sano Mono? https://dejavu-fonts.github.io/

I'm on linux and Fontlab is only available onWindows and Mac. I have FontForge installed and I'm already struggling at the first step adding a suffix to each character in a font.

That python script that is mentioned on the "apply to other fonts" part though.. `For example, using the addfeatures.py script. ` https://github.com/simoncozens/test-fonts/blob/master/addfeatures.py That looks like a simple script and Chatgpt had a plan how to use it once I first got the bold glyphs copied to the regular font file.

I then tried to keep on getting the glyphs copied. Chatgpt gave me instructions on copying them with fontforge, but the options were not there or they were grayed out. Then Chatgpt tried instructing me to do it with a python script using fonttools, but after 30+ minutes with trial and error, we couldn't get them copied. I also tried asking Gemini. So apparently this is a too difficult thing for a LLM chatbot for now at least

**Comment by khlsvr** [comment]
Just an update where I have got so far.

Using Fontforge I got so far (hopefully right way) that I added new encoding slots to regular font file, then pasted these bold glyphs from bold font file to the new empty slots in regular font file. Then manually.. sigh.. renamed all of them to a.bold A.bold b.bold B.bold etc. At this point I guess I have the font file with regular and both in one.

Now after this I should paste the contents of the .fea file to a proper place and I'm struggling.

I tried going to Element>font info> Lookups> Add Lookup. Choosing in new box "contextual substitition > dropdown "Calt" contextual substition. Clicking the box opens a window that has DFLT{dflt} latn{dflt,CAT, ESP ,GAL ...} ends in math{dflt}.

I tried then pasting the contents of the .fea (or rather the simplified A-Z code snippet) from the fontlab tutorial to it overwriting those or also just below them, but I'm getting "script tag on line 1 (calt) is too long. It may be at most 4 letters"

I might try consulting fontforge people or read more how to do the calt stuff there but now I have run out of time for today probably.
